ipfixprobe.spec.in

```diff
@@ -57,8 +57,8 @@ Vendor: CESNET, z.s.p.o.
 Packager: @USERNAME@ <@USERMAIL@>
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}
 Summary: IPFIX flow exporter with various extending IPFIX elements exported by plugins.
-Requires: libatomic
-BuildRequires: gcc gcc-c++ make doxygen pkgconfig libatomic
+Requires: libatomic fuse3
+BuildRequires: gcc gcc-c++ make doxygen pkgconfig libatomic telemetry
```

Comment:
This should be checked again - BuildRequires has no fuse3-devel or something like that; Requires has no telemetry - is there anything needed?

Reply:
For ubuntu there is libfuse3-dev installed...

Reply:
fixed